macOS/DuckDuckGo/Preferences/View/PreferencesAboutView.swift

```diff
@@ -48,6 +48,10 @@ extension Preferences {
                     UpdatesSection(areAutomaticUpdatesEnabled: $areAutomaticUpdatesEnabled, model: model)
 #endif
                 }
+            }.task {
+                if model.mustCheckForUpdatesBeforeUserCanTakeAction {
+                    model.checkForUpdate()
+                }
```

Comment:
When the feature flag is ON we may now check for new updates.

Reply:
Note that `model.checkForUpdate()` calls `updater.checkForUpdateSkippingRollout()`, which bypasses rollout group, so it is a behavior change. 

I.e. Currently we only skips rollout group when the user explicitly clicks on the Check for Updates button, with this change if the user lands on Preferences > About, we immediately pulls the most recent update.